doc/users/next_whats_new/boldsym_mathtext.rst

```diff
@@ -0,0 +1,12 @@
+Boldsymbol mathtext command ``\boldsymbol``
+-----------------------------------------------------
+
+Supports using the ``\boldsymbol{}`` command in mathtext:
+
+To change font to bold and italic enclose the text in a font command as
+shown:
+
+.. code-block::
```

Comment:
I do not fully get this warning, but I guess it must be fixed one way or the other. Maybe better to add a figure in the other PR, like `plt.text(...)` and then add the alternative version in that figure here?

Reply:
Currently, I am getting these results from the current implementation. Should I add the `mathbfit` to the other PR, and the `boldsymbol` to the current one?

![image](https://user-images.githubusercontent.com/35831931/231386263-0014a9ba-62f2-49c4-822b-a182b270ce56.png)

Reply:
I think it is just missing a blank line.

```
.. code-block::

   r'$\boldsymbol{a+2+\alpha}$'
```

(note also that the closing quote is missing anyway)